src/Resolvers/Microsoft.NET.Sdk.WorkloadManifestReader/IWorkloadManifestProvider.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace Microsoft.Net.Sdk.WorkloadManifestReader
+{
+    public interface IWorkloadManifestProvider
+    {
+        IEnumerable<Stream> GetManifests();
```

Comment:
Multiple manifests are possible?

Reply:
this abstracts the process of collecting a set of workload manifests to be composed together e.g. all the workload manifests for a given sdk band

it could certainly be named better